packages/office-ui-fabric-react/src/components/Autofill/BaseAutofill.tsx

```diff
@@ -0,0 +1,284 @@
+import * as React from 'react';
+import { IBaseAutofillProps, IBaseAutofill } from './BaseAutofill.types';
+import {
+  BaseComponent,
+  KeyCodes,
+  autobind,
+  getNativeProps,
+  inputProperties
+} from '../../Utilities';
+
+export interface IBaseAutofillState {
+  displayValue?: string;
+}
+
+const SELECTION_FORWARD = 'forward';
+const SELECTION_BACKWARD = 'backward';
+
+export class BaseAutofill extends BaseComponent<IBaseAutofillProps, IBaseAutofillState> implements IBaseAutofill {
```

Comment:
Should these be changed to AutoFill also?

Reply:
Maybe, that's a good idea.